frontend/public/actions/k8s.ts

```diff
@@ -21,7 +21,7 @@ export {
   receivedResources,
 } from '@console/dynamic-plugin-sdk/src/app/k8s/actions/k8s';
 
-export const API_DISCOVERY_POLL_INTERVAL = 60000;
+export const API_DISCOVERY_POLL_INTERVAL = 60_000 * 5;
```

Comment:
Is it possible to define this constant in one place instead of two?

Reply:
Yeah, I'll try to put this somewhere that makes sense.